assets/js/modules/analytics/util/index.js

```diff
@@ -462,19 +461,6 @@ export const getTopPagesReportDataDefaults = () => {
 		},
 	];
 
-	if ( getModulesData().analytics.settings.adsenseLinked ) {
```

Comment:
If we're not using this, should we not also remove the setting and all of the added filters, etc around it?

Reply:
Potentially, I thought about that too. I think we can keep it for now and revisit separately. We should think about potential other use-cases for that connection and see if we still need it then.